packages/react-magma-dom/src/components/Combobox/Combobox.tsx

```diff
@@ -44,6 +44,7 @@ export function InternalCombobox<T>(props: ComboboxInterface<T>) {
     onIsOpenChange,
     onItemCreated,
     placeholder,
+    renderItem,
```

Comment:
We have a `components` object for swapping out default implementations of components. Is there a reason to have this in a separate prop? Seems like `components` would cover this and all future custom component needs.